### PR TITLE
feat(navigation): introduce new parameter to allow hiding the busines…

### LINKED ISSUE #175: Add parameter to hide business partner selection
To support some cases like Merchant Center Profile where at some points there's no business partner context we should allow to hide the button for an application via a new parameter

### PR DESCRIPTION
**Pull request by dami-gg**
…s partner

Closes #175

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
   devServer: {
     contentBase: __dirname + '/demo'
   },
-  devtool: 'inline-source-map',
+  devtool: 'eval',
```
Comment (frontendherodk) [inline webpack.config.js]: I would be hesitant to push this since we don't differentiate between dev and pro config

Comment (dami-gg) [inline webpack.config.js]: true, I committed this by mistake. I usually enable it to work locally

**Comment by dami-gg** [comment]
@frontendherodk Fixed!

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**